src/main/java/fr/xephi/authme/commands/AdminCommand.java

```diff
@@ -97,7 +103,11 @@ public boolean onCommand(CommandSender sender, Command cmnd, String label,
 
         if (args[0].equalsIgnoreCase("purge")) {
             if (args.length != 2) {
-                sender.sendMessage("Usage: /authme purge <DAYS>");
+                sender.sendMessage("Usage: /authme purge <days>");
+                return true;
+            }
+            if (Integer.parseInt(args[1]) < 30) {
```

Comment:
I do agree, I got once my database full purged by an admin that did /authme purge 1, as he tought he was working inversly (would only clear data till 1 day before).
This is a Safe measue for mistype.
You may want to set this to one week or one day if issued by console ;)